demos/rain_prediction.ipynb

```diff
@@ -174,23 +177,14 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "fs.put('data/weather-2010s.json',  f'{REPO_NAME}/{NEW_BRANCH_NAME}/weather-2010.json')"
+    "fs.put('data/weather-2010s.json',  f'{REPO_NAME}/{NEW_BRANCH_NAME}/weather-2010.json', precheck=True)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "We can circumvent this by disabling pre-checking file checksums on a specific put operation. We do this by passing `precheck=False` to the `PUT` operation."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "fs.put('data/weather-2010s.json',  f'{REPO_NAME}/{NEW_BRANCH_NAME}/weather-2010.json', precheck=False)"
+    "However, expecially when you deal with large files the precheck makes sense to avoid repeated but unnecessary transfer of large files. "
```

Comment:
```suggestion
    "However, especially when dealing with large files, using prechecks makes sense to avoid repeated unnecessary transfers of large files. "
```